src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/ArrivalTransformer.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.health.api.model.Arrival
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.ArrivalEntity
+
+@Component
+class ArrivalTransformer() {
+  fun transformJpaToApi(jpa: ArrivalEntity?): Arrival? {
```

Comment:
This looks like very java way of doing things `if (jpa == null) return null`

Should just be able to do something like this ` fun transformJpaToApi(jpa: ArrivalEntity?): Arrival? = jpa?.let { Arrival( ... ) }`

Reply:
What Mike says above actually what I would also do.

Reply:
Have switched these to expression bodied for now, hands are a bit tied on adding a constructor to the API models since they're generated but will go for that approach if we switch away from Open API Generator